test/Microsoft.AspNetCore.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -972,16 +972,15 @@ public void WriteTo_WritesFormattedTagHelper(TagHelperOutput output, string expe
             Assert.Equal(expected, writer.ToString(), StringComparer.Ordinal);
         }
 
-        // This tests a separate code path that's used by THO when the writer is an HtmlTextWriter.
-        // The output should be the same, but we do some specific perf optimizations on this path.
         [Theory]
         [MemberData(nameof(WriteTagHelper_InputData))]
-        public void WriteTo_WritesFormattedTagHelper_HtmlTextWriter(TagHelperOutput output, string expected)
+        public void CopyTo_CopiesToBuilder(TagHelperOutput output, string expected)
```

Comment:
Add either a test that you can call `CopyTo()` twice and get the same result or a test that confirms `Attributes.Count` before equals `Attributes.Count` after.